### PR TITLE
Add capability to use additionalModelTypeAnnotations

### LINKED ISSUE #85: How to use additionalModelTypeAnnotations
I would like to include some extra additional annotations to the models generated here.
I come from openapi-generator project and there is a way of doing it using a property called additionalModelTypeAnnotations.

I’ve done a bit of digging and I found same property mentioned on templates here as well but I wasn’t able to find the way of adding configuration for it.

Is it possible? If it is not I think that would be a good addition to the project.

Thank you

### PR DESCRIPTION
**Pull request by miguelchico**
Initial work to fix #85

### CODE REVIEW

File: deployment/src/main/resources/templates/additionalModelTypeAnnotations.qute [hunk 1]
```diff
@@ -1 +1,3 @@
-{#for annon in m.additionalModelTypeAnnotations.orEmpty}{annon}{/for}
```
Comment (miguelchico) [inline deployment/src/main/resources/templates/additionalModelTypeAnnotations.qute:1]: I'm not an expert in using Qute so I'm not sure if this change can be done in a different way, I'm open to suggestions

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
@fjtirado can you please take a look? I'm planning to merge this on Monday and release a new version for @miguelchico 

@miguelchico I'm assuming this PR is ready? Looks ready to me. :)

**Comment by miguelchico** [comment]
@ricardozanini it is ready in my opinion. It would be great to get the new version next week. Thank you!

File: deployment/src/main/resources/templates/additionalModelTypeAnnotations.qute (hunk 1 above)
Comment (fjtirado) [reply deployment/src/main/resources/templates/additionalModelTypeAnnotations.qute:1]: Me enither, but Im guessing what happens if you try with 
{additionalModelTypeAnnotations.orEmpty}

**Comment by ricardozanini** [comment]
@miguelchico congrats! https://github.com/quarkiverse/quarkus-openapi-generator/releases/tag/0.8.0

Please take a few hours for Maven Central to update the artifacts.

**Pull request merged**